crates/sncast/tests/data/scripts/old_sncast_std/scripts/src/map_script.cairo

```diff
@@ -0,0 +1,59 @@
+use sncast_std::{
+    declare, deploy, invoke, call, DeclareResult, DeployResult, InvokeResult, CallResult, get_nonce
+};
+
+fn second_contract() {
```

Comment:
I'm afraid this can lead to race condition with the other `general` script that does exactly the same, with the same contracts (and two identical contracts cannot be declared - it works fine when your test executes second as you do not check the result, but if your script will ever get executed first I guess the other test will fail, because we check its execution result).

I think contract declaration (and maybe even deployment) is not essential to the test that uses this script, so let's maybe just remove it?

Reply:
Done